### PR TITLE
Add tox

### PR DESCRIPTION
**Pull request by sfdye**

### CODE REVIEW

File: tests/Logging_.py [hunk 1]
```diff
@@ -61,31 +61,14 @@ def testLoggingWithBasicAuthentication(self):
         self.assertEqual(
             github.Github(self.login, self.password).get_user().name, "Vincent Jacques"
         )
-        # In Python 3.4+, dicts are not output in the same order as in Python 2.7.
```
Comment (s-t-e-v-e-n-k) [inline tests/Logging_.py]: Bad! This is unrelated. :-) However, I noticed this monstrosity thanks to this, so I've pushed up https://github.com/PyGithub/PyGithub/pull/1315. I've run black over my changes, so it shouldn't impact your black CI changes.

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[tox]
+envlist = py27,py36
```
Comment (s-t-e-v-e-n-k) [inline tox.ini]: Since we also support py35, py37 and py38 in travis, please add them here too.

Comment (sfdye) [reply tox.ini]: This PR is still WIP 🙂

File: tests/Logging_.py (hunk 1 above)
Comment (s-t-e-v-e-n-k) [reply tests/Logging_.py]: @sfdye Shall I merge in that refactoring and you can rebase this to not lose any test coverage?

Comment (sfdye) [reply tests/Logging_.py]: 👍

**Comment by s-t-e-v-e-n-k** [comment]
@sfdye Do you have time to come back to this, or shall I work on it?

**Comment by sfdye** [comment]
Let me close this and give it another shot in a new PR.

**Pull request closed without merging**